src/platform/linux/wayland.h

```diff
@@ -150,36 +125,32 @@ namespace wl {
 
   public:
     enum interface_e {
-      XDG_OUTPUT,  ///< xdg-output
-      WLR_EXPORT_DMABUF,  ///< Export dmabuf
-      MAX_INTERFACES,  ///< Maximum number of interfaces
+      XDG_OUTPUT,         ///< xdg-output
+      WLR_EXPORT_DMABUF,  ///< screencopy manager
+      LINUX_DMABUF,       ///< linux-dmabuf protocol
+      MAX_INTERFACES
```

Comment:
```suggestion
      XDG_OUTPUT,  ///< xdg-output
      WLR_EXPORT_DMABUF,  ///< screencopy manager
      LINUX_DMABUF,  ///< linux-dmabuf protocol
      MAX_INTERFACES,  ///< Maximum number of interfaces
```

We don't use aligned comments. I also know the final comment is redundant; however the docs will fail without a comment here.